### PR TITLE
EVMC: do not map unknown non-critical errors as internal errors

### PR DESCRIPTION
**Pull request by axic**
Addresses https://github.com/ewasm/tests/pull/52#issuecomment-408276981

### CODE REVIEW

File: libevm/EVMC.cpp [hunk 1]
```diff
@@ -73,8 +73,14 @@ owning_bytes_ref EVMC::exec(u256& io_gas, ExtVMFace& _ext, const OnOpFunc& _onOp
         cwarn << "Execution rejected by EVMC, executing with default VM implementation";
         return VMFactory::create(VMKind::Legacy)->exec(io_gas, _ext, _onOp);
 
+    case EVMC_INTERNAL_ERROR:
```
Comment (chfast) [inline libevm/EVMC.cpp]: Why did you added this case?

Comment (axic) [reply libevm/EVMC.cpp]: Only for clarity for the reader, but it is not necessary of course.

Comment (axic) [reply libevm/EVMC.cpp]: Should we have a comment here that negative ones are internal errors while positive ones are non-internal?

**Comment by axic** [comment]
Merging this now to have it fixed (needed by ewasm tests). This could be improved/changed in the future.

**Pull request merged**